File: TVSeriesAPI/Interfaces/IActors.cs
using System.Collections.Generic;

namespace TVSeriesAPI.Interfaces
{
  public interface IActors
  {
    string Id { get; set; }
    string Name { get; set; }
    string Image { get; set; }
    string Age { get; set; }
    string Country { get; set; }
    List<string> InSeries { get; set; }
  }

}

File: tvseries/src/interfaces/IActors.ts
import { IInMovies } from "./IInMovie";
import { IInSeries } from "./IInSeries";
export interface IActors {
  id?: string;
  name: string;
  image?: string;
  age: string;
  country: string;
  inSeries?: IInSeries[];
  inMovies?: IInMovies[];
}
